test/fixtures/subscription-generator/app/schedule/interval.js

```
@@ -0,0 +1,16 @@
+
+class Interval extends Subscription {
+  static get schedule() {
+    return {
+      type: 'worker',
+      interval: 4000,
+    };
+  }
+
+  * subscribe() {
+    this.ctx.logger.info('interval');
+  }
```

Biome (1.9.4)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)